ocs_ci/ocs/node.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import print_function
+
+import logging
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+log = logging.getLogger(__name__)
+
+
+def get_node_objs(node_names=None):
+    """
+    Get node objects by node names
+
+    Args:
+        node_names (list): The node names to get their objects for.
+            If None, will return all cluster nodes
+
+    Returns:
+        list: Cluster node OCP objects
+
+    """
+    nodes_obj = OCP(kind='node')
+    node_objs = nodes_obj.get()['items']
+    if not node_names:
+        return node_objs
+    else:
+        return [
+            node_obj for node_obj in node_objs if (
+                node_obj.get('metadata').get('name') in node_names
+            )
+        ]
+
+
+def wait_for_nodes_status(node_names, status='Ready'):
+    """
+    Wait until all nodes are in Ready status
+
+    Args:
+        node_names (list): The node names to wait for to be in Ready state
+        status (str): The node status to wait for
+            (e.g. 'Ready', 'NotReady', 'SchedulingDisabled')
+
+    Returns:
+        bool: True if all nodes reached status Ready, False otherwise
```

Comment:
```
        bool: True if all nodes reached status Ready, False otherwise
```
Consider other status as mentioned in
status (str): The node status to wait for
            (e.g. 'Ready', 'NotReady', 'SchedulingDisabled')

Reply:
good catch